CHANGELOG.rst

```diff
@@ -15,18 +15,18 @@ v0.7.0 RC2
 
 * Subvoxel R2S allows activation and photon source sampling by
   material within a mesh voxel
```

Comment:
I think that if the above is on a single line, the formatting might work better.

Reply:
on it!